site/_includes/explanations/partitioned-topic-admin.md

```diff
@@ -1,4 +1,4 @@
-You can use Pulsar's [admin API](../../admin/AdminInterface) to create and manage partitioned topics.
+You can use Pulsar's [admin API](../../admin-api/overview) to create and manage partitioned topics.
```

Comment:
Actually, looking at it more carefully, this link is on the top of the page with the list of all the "partitioned topics" APIs, so there's no need to link back to itself. 

I think that linking back to the "overview" page is correct in these cases. (to give instructions on how to setup the tools, etc..)